blog/management/commands/bootstrap_testing_server.py

```
@@ -0,0 +1,93 @@
+from django.contrib.auth import get_user_model
+from django.core.management import call_command
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+
+from blog.models import Post
+
+FIXTURE_PATH = Path(__file__).resolve().parents[2] / "fixtures" / "initial_data.json"
+
+TEST_USERNAME = "testing"
+TEST_EMAIL = "testing@testing.com"
+TEST_PASSWORD = "testing"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "TEST_PASSWORD"

(S105)